src/editors/containers/EditorContainer/index.jsx

```diff
@@ -64,7 +64,7 @@ export const EditorContainer = ({
           />
         </div>
       </ModalDialog.Header>
-      <ModalDialog.Body className="pb-6">
+      <ModalDialog.Body className="pb-6 min-vh-100">
```

Comment:
Why was this added? I feel like it might be an issue for full screen modal stuff?

Reply:
Without this, the dropdown can get cut off because the "body" of the editor doesn't extend all the way down